server.js

```diff
@@ -1,10 +1,28 @@
 const express = require("express");
+const cors = require('cors')
+const pino = require('express-pino-logger')();
+const axios = require('axios')
+require('dotenv').config()
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
+app.use(cors())
```

Comment:
This will allow any origin to make a request against our server. We want to be specific on what origins we allow. Pass in the allowed origin as an environment variable, and give it a default of 'http://localhost:3000'. See how I've done the PORT as an example.